smart_open/tests/test_smart_open.py

```diff
@@ -1621,6 +1635,46 @@ def test(self):
         self.assertEqual(expected, actual)
 
 
+class GetBinaryModeTest(parameterizedtestcase.ParameterizedTestCase):
+    @parameterizedtestcase.ParameterizedTestCase.parameterize(
+        ('mode', 'expected'),
+        [
+            ('r', 'rb'),
+            ('r+', 'rb+'),
+            ('rt', 'rb'),
+            ('rt+', 'rb+'),
+            ('r+t', 'rb+'),
+            ('w', 'wb'),
+            ('w+', 'wb+'),
+            ('wt', 'wb'),
+            ('wt+', 'wb+'),
+            ('w+t', 'wb+'),
+            ('a', 'ab'),
+            ('a+', 'ab+'),
+            ('at', 'ab'),
+            ('at+', 'ab+'),
+            ('a+t', 'ab+'),
+        ]
+    )
+    def test(self, mode, expected):
+        actual = smart_open.smart_open_lib._get_binary_mode(mode)
+        assert actual == expected
+
+    @parameterizedtestcase.ParameterizedTestCase.parameterize(
+        ('mode', 'expected'),
```

Comment:
I'm not familiar with `ParameterizedTestCase `, but why bother declaring two fields and then supply only one?

Reply:
You're right, that's copypasted from the test above. Will fix.